src/Subscriptions/Storage/RedisStorageManager.php

```diff
@@ -38,7 +38,11 @@ public function __construct(ConfigRepository $config, RedisFactory $redis)
         $this->connection = $redis->connection(
             $config->get('lighthouse.subscriptions.broadcasters.echo.connection') ?? 'default'
         );
-        $this->ttl = $config->get('lighthouse.subscriptions.storage_ttl');
+        $ttl = $config->get('lighthouse.subscriptions.storage_ttl');
+        if (is_string($ttl)) {
+            $ttl = (int) $ttl;
+        }
+        $this->ttl = $ttl;
```

Comment:
We can directly assign if the value `is_int()`. If it `is_string()`, we should also check it `is_numeric()` before casting, in all other cases we should throw and indicate the value is neither an int or a numeric string.

Reply:
Good idea! Done